src/site/content/en/mini-apps/an-example-project/index.md

```diff
@@ -0,0 +1,256 @@
+---
```

Comment:
Needs a new URL. We need some kind of common prefix. `web.dev/mini-app-example-project`?

Reply:
Used "mini-app" throughout.